tests/lit-tests/loop_nounroll.ispc

```diff
@@ -0,0 +1,99 @@
+// Test to check '#pragma nounroll' functionality for different loop statemnts.
+
+// RUN: %{ispc} %s --target=avx2-i32x8 --nowrap -O0 --emit-llvm-text -o - | FileCheck %s -check-prefix=CHECKO0
+// RUN: %{ispc} %s --target=avx2-i32x8 --nowrap -O2 --emit-llvm-text -o - | FileCheck %s -check-prefix=CHECKO2
+// REQUIRES: X86_ENABLED
+
+// CHECKO0: br label %for_test, !llvm.loop
```

Comment:
Label names are not reliable. I suggest wildcarding the label names.

Reply:
Done